src/markdown/ciceromark@0.3.0.cto

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace org.accordproject.ciceromark
+
+import org.accordproject.commonmark.Node from https://models.accordproject.org/markdown/commonmark@0.2.0.cto
+import org.accordproject.commonmark.Child from https://models.accordproject.org/markdown/commonmark@0.2.0.cto
+
+/**
+ * A model for Accord Project extensions to commonmark
+ */
+
+abstract concept Element extends Child {
+  o String name
+  o String elementType optional
```

Comment:
`elementType` is the type of the element (this is the type of the variable, clause, etc). I didn't use type because it conflicts with `type` of lists from commonmark AST (which is a little annoying, but I wanted to keep the commonmark changes as minimal as possible).